siliconcompiler/tools/klayout/screenshot.py

```diff
@@ -21,6 +21,11 @@ def setup(chip):
     chip.set('tool', tool, 'task', task, 'script', step, index, script, clobber=clobber)
     chip.set('tool', tool, 'task', task, 'option', step, index, option, clobber=clobber)
 
+    pdk = chip.get('option', 'pdk')
+    stackup = chip.get('option', 'stackup')
+    if chip.valid('pdk', pdk, 'var', 'klayout', 'hide_layers', stackup):
+        scrot_hide_layers = chip.get('pdk', pdk, 'var', 'klayout', 'hide_layers', stackup)
```

Comment:
scrot_hide_layers -> hide_layers?

Reply:
Yeahh, I followed the OpenROAD script naming conventions here, but it does seem like a bit of an odd choice for an abbreviation.

Reply:
If that came from the OR script, we should probably fix it there too. I don't think I picked a good name.

Reply:
In any case, I made it `layers_to_hide` here and in `show.py`.